jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -1332,6 +1343,19 @@ Function Start-Migration {
                 Throw [System.Management.Automation.ValidationMetadataException] "You must supply a value for JumpCloudAPIKey when autobinding a JC User"
                 break
             }
+            # Throw error if $ret is false, if we are autobinding users and the specified username does not exist, throw an error and terminate here
+            $ret, $JumpCloudUserId, $JumpCloudUsername, $JumpCloudsystemUserName = Test-JumpCloudUsername -JumpCloudApiKey $JumpCloudAPIKey -JumpCloudOrgID $JumpCloudOrgID -Username $JumpCloudUserName
+            # Write to log all variables above
+            Write-ToLog -Message:("Ret = $($ret) , JumpCloudUserName: $($JumpCloudUserName) , JumpCloudUserId: $($JumpCloudUserId) , JumpCloudsystemUserName = $($JumpCloudsystemUserName)")
+
+            if($JumpCloudsystemUserName){
+                $JumpCloudUsername = $JumpCloudsystemUserName
+            }
+            if ($ret -eq $false) {
+                Throw [System.Management.Automation.ValidationMetadataException] "The specified JumpCloudUsername does not exist"
```

Comment:
```suggestion
                Write-ToLog -Message:("The specified JumpCloudUsername does not exist, migration will not continue")
                Throw [System.Management.Automation.ValidationMetadataException] "The specified JumpCloudUsername does not exist"
```

Reply:
If we add a `Write-ToLog` message here it'll show up in our verbose logs. I was a bit confused looking at the pester tests since not all of the output is gathered. This suggestion is more for sanity sake.